### DISCUSSION TITLE
Incorrect towing capcity

### DISCUSSION THREAD

**Discussion by avanmeer** [iamabrom/towcalculator.app#1] [General]
You have towing capacity as follows:
GCWR - GVWR (assumes payload is at max)

Towing capacity is GCWR - curbWeight.

Am I wrong, or missing something?

**Comment by iamabrom** [comment]
Hi @avanmeer, technically the "Towing capacity" (or tow rating) is GCWR - curb weight.

However, I choose to display GCWR - GVWR and I indicate this makes the assumption that the tow vehicle is at max payload. The reality is towing capacity without factoring in payload is a meaningless and arbitrary number. No one will ever tow something without payload in the tow vehicle- there at least has to be a person driving the vehicle.

The real Tow Capacity in the calculator is at the bottom of the web app under the "GCWR and Tow Capacity"
